Update home.tpl

andria-fyi opened the pull request:
- Adds homepage tiles  and images for Balancer, Clean and Green, and Votewise.
- Updates
  - PAX homepage tile
  - Formatting to clarify element parentage
- Removes:
  - Excess commented-out code
  - PAWS project tile
  - Plus icons in the project tiles grid

andria-fyi committed:
Update home.tpl

themightychris reviewed (changes requested):
there's a parameter error in the new projcet

Commit:
Update projects and modals on homepage

Commit:
Remove excess commented out code

These are mostly noise and if we ever need to refer to these we can look back through git

Commit:
Format first section for clarity

Commit:
Format project templates for clarity

Commit:
Format modal section for clarity

Commit:
Remove plusses from grid

Commit:
Fix quote error in balancer project modal

Commit:
Add Instagram link to page

Commit:
Use actual Balancer logo
